wiki/Gameplay/Notelock/de.md

```diff
@@ -0,0 +1,32 @@
+---
+tags:
+  - note lock
+  - jittering notes
+  - shaking notes
+---
+
+# Notelock
+
+![](img/notelock.gif "Ein Beispiel für Notelock: die zweite Note ist unzugänglich")
+
+**Notelock**, oder **note lock**, ist ein informeller Begriff für eine Spielmechanik von [osu!](/wiki/Game_mode/osu!) welche einen Spieler daran hindern kann, ein Hit-Objekt zu treffen. Dies geschieht, wenn **zwei** Bedingungen gleichzeitig erfüllt sind:
+
+1. Die [Zeitfenster](/wiki/Beatmapping/Overall_difficulty#timing) zweier Hit-Objekte überschneiden sich.
+2. Das erste dieser beiden Objekte wurde noch nicht beurteilt (getroffen oder verfehlt).
+
+In diesem Fall wird das zweite Objekt hinter dem ersten *gesperrt*, was osu! veranlasst, die Eingaben des Spielers zu ignorieren, bis das Trefferfenster des ersten Objekts verstrichen ist. Die Notsperre wird nur dann zu einem Problem, wenn der Spieler nicht in der Lage ist, zum vorherigen Objekt zurückzukehren und mit ihm zu interagieren, wodurch die Sperre aufgehoben wird. In diesem Fall kann eine gesperrte Note einen kaskadierenden Fehler verursachen, der den Spieler dazu zwingt, mehr und mehr nachfolgende Objekte zu verpassen, bis er keine [Gesundheit](/wiki/Beatmapping/Health) mehr hat und die Beatmap nicht besteht.
```

Comment:
"einen kaskadierenden Fehler verursachen" -> "einen Lawineneffekt auslösen"
(for easier comprehension)